app/_src/kubernetes-ingress-controller/troubleshooting.md

```diff
@@ -310,58 +310,134 @@ the Helm chart), wait for the Deployment to restart, run `kubectl
 port-forward <POD_NAME> 10256:10256`, and visit `http://localhost:10256/debug/pprof/`.
 
 {% if_version gte:2.8.x %}
```

Comment:
This will also display for 2.8.x, so I wasn't sure if your changes were specific to 2.9.x and later.